prereise/gather/demanddata/eia/map_ba.py

```diff
@@ -95,3 +102,56 @@ def map_buses_to_county(bus_county_map):
         except TypeError:
             bus_no_county_match.append(index)
     return bus_county_map, bus_no_county_match
+
+
+def map_buses_to_ba(bus_df):
+    """Find the Balancing Authority in the U.S. territory that each query bus belongs to
+    based on GIS information.
+
+    :param (*pandas.DataFrame*) bus_df: data frame contains a list of entries with
+        lat and long of buses.
+    :return: (*tuple*) -- the first entry is the input data frame with two columns,
+        "County" and "BA", added for each bus and the second entry is the list of bus
+        indices that no county matches based on Census API (counties of such buses are
+        assigned based on its nearest neighbour).
+    """
+
+    bus_ba_map, bus_no_county_match = map_buses_to_county(bus_df)
+    # hard coded fix for county name mismatch between the returns from census API and
+    # BA_County_map.json
+    bus_ba_map.County.replace(
+        {
+            "Danville City__VA": "Danville__VA",
+            "Harrisonburg City__VA": "Harrisonburg__VA",
+            "Franklin City__VA": "Franklin__VA",
+        },
```

Comment:
Do you think it is worth putting this in a **const.py** file?

Reply:
Given it doesn't have a concrete meaning outside this function, I would suggest to keep it here.